src/main/java/org/fogbowcloud/manager/core/ManagerController.java

```diff
@@ -1,85 +1,47 @@
 package org.fogbowcloud.manager.core;
 
-import java.util.List;
 import java.util.Properties;
-import java.util.TimerTask;
-import java.util.concurrent.Executors;
 
 import org.apache.log4j.Logger;
 import org.fogbowcloud.manager.core.constants.ConfigurationConstants;
-import org.fogbowcloud.manager.core.constants.DefaultConfigurationConstants;
-import org.fogbowcloud.manager.core.datastore.ManagerDatastore;
 import org.fogbowcloud.manager.core.instanceprovider.InstanceProvider;
-import org.fogbowcloud.manager.core.models.orders.Order;
-import org.fogbowcloud.manager.core.models.orders.OrderState;
-import org.fogbowcloud.manager.core.models.orders.instances.OrderInstance;
+import org.fogbowcloud.manager.core.models.orders.OrderRegistry;
```

Comment:
This class does not exist anymore.

Reply:
This classes in reds lines don't exist in my branch, only the one identified in the green line. 
The codes in reds lines might belong to the branch create-compute.